components/brave_sync/brave_profile_sync_service_impl.h

```diff
@@ -32,6 +32,7 @@ FORWARD_DECLARE_TEST(BraveSyncServiceTest, ClientOnGetInitData);
 FORWARD_DECLARE_TEST(BraveSyncServiceTest, OnGetInitData);
 FORWARD_DECLARE_TEST(BraveSyncServiceTest, OnSaveBookmarksBaseOrder);
 FORWARD_DECLARE_TEST(BraveSyncServiceTest, OnCompactComplete);
+FORWARD_DECLARE_TEST(BraveSyncServiceTest, OnRecordsSent);
```

Comment:
BraveSyncServiceTest doesn't contain the test named `OnRecordsSent`, looks the actial test is done in `(BraveSyncServiceTest, ExponentialResend)`
So this line is  not required.

Reply:
addressed in 73b682f8175917b4bd8e2356088acb0af9924217